src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -243,7 +237,7 @@ bool emberAfGroupsClusterViewGroupCallback(app::CommandHandler * commandObj, con
         VerifyOrExit((writer = commandObj->GetCommandDataIBTLVWriter()) != nullptr, err = CHIP_ERROR_INCORRECT_STATE);
         SuccessOrExit(err = writer->Put(TLV::ContextTag(0), status));
         SuccessOrExit(err = writer->Put(TLV::ContextTag(1), groupId));
-        SuccessOrExit(err = writer->PutString(TLV::ContextTag(2), groupName.data(), static_cast<uint32_t>(groupName.size())));
+        SuccessOrExit(err = writer->PutString(TLV::ContextTag(2), mapping.name, static_cast<uint32_t>(nameSize)));
```

Comment:
That seems to indicate that names are supported, but they are missing, length will be zero.

How do we tell the difference between an empty name, and name storage not supported (i.e. writing one would fail) ?

Reply:
According to the spec, section 1.3.6.8.1.:
```
 If the status is SUCCESS, and group names are supported, the GroupName field is set to the Group Name associated with that group identifier in the Group Table; otherwise it is set to the empty string.
```